modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -67,14 +68,8 @@ public function is_iframed_block_editor() {
 	 */
 	public function show_error_if_logged_out() {
```

Comment:
We probably want to rename this function and its docstring.

Reply:
Actually, we don't need this function at all since this is the default behavior of WordPress: if redirects the user to the login page with the URL they are attempting to access as `redirect_to` param.

Reply:
Removed in 537b61b3a9f748bcaaf3cbb6d71447f3404f5f5b